### PR TITLE
feat: :sparkles: Added new TypeBundle pkg

### PR DESCRIPTION
**Pull request by timbrinded**
#### Added:
- New `types-bundle` package
  - This is the new single point for our manual RPC/ runtimeApi definitions
- New workflows for publishing to NPM the `@storagehub/api-augment` and `@storagehub/types-bundle` packages
 
#### Changed:
- Refactored `api-augment` to import from `types-bundle`
- Updates to generated ApiAugment (hasn't been run in main )
- Workspace changes to _fearlessly_ link between differently packages in monorepo
- Package updates (plus a workaround for zombienet because it breaks)

### CODE REVIEW

File: .github/workflows/parachain.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Storage Hub: Parachain CI/CD Workflow
+ob# Storage Hub: Parachain CI/CD Workflow
```
Comment (ffarall) [inline .github/workflows/parachain.yml:1]: Is there a typo here?

File: .github/workflows/api-augment-publish.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+name: Publish api-augment to npm.js
```
Comment (ffarall) [inline .github/workflows/api-augment-publish.yml:1]: Why do we need to publish `api-augmet`? I thought with `types-bundle` it was enough for our pjs fork.

File: test/util/bspNet/api.ts [hunk 3]
```diff
@@ -7,36 +6,17 @@ import type { KeyringPair } from "@polkadot/keyring/types";
...
   const baseApi = await ApiPromise.create({
     provider: new WsProvider(uri),
     noInitWarn: true,
-    types,
-    rpc: rpcMethods,
-    runtime
+    typesBundle: BundledTypes
+    // types,
+    // rpc: rpcMethods,
+    // runtime
```
Comment (ffarall) [inline test/util/bspNet/api.ts:19]: Maybe remove this if not needed?

File: README.md [hunk 4]
```diff
@@ -60,3 +60,30 @@ Full instructions can be found [here](test/README.md#local-usage).
...
+
+Any new Structs or ErrorEnums can be defined at `/src/types.ts` , using existing examples for guidance, and any new branded types as well.
+
+Afterwards in the directory, run:
+
+```sh
+pnpm i 
+pnpm build
+pnpm fmt:fix
+```
```
Comment (ffarall) [inline README.md:78]: Can we bundle this into `pnpm typegen` so that it is quicker/easier to remember for devs?

**Review by ffarall — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (timbrinded) [reply README.md:78]: Done

File: .github/workflows/api-augment-publish.yml (hunk 2 above)
Comment (timbrinded) [reply .github/workflows/api-augment-publish.yml:1]: types bundle only gives RPC and RuntimeAPI methods.

You need apiAugment to show all autoscraped query, consts and extrinsic calls

File: .github/workflows/parachain.yml (hunk 1 above)
Comment (timbrinded) [reply .github/workflows/parachain.yml:1]: done

File: test/util/bspNet/api.ts (hunk 3 above)
Comment (timbrinded) [reply test/util/bspNet/api.ts:19]: done

**Pull request merged**